neo4j/spatial/__init__.py

```diff
@@ -72,22 +72,6 @@ def dehydrate_point(value):
     return _hydration.dehydrate_point(value)
 
 
-# TODO: 6.0 remove
-@deprecated(
-    "hydrate_point is considered an internal function and will be removed in "
-    "a future version"
-)
-@wraps(_hydration.dehydrate_point)
-def dehydrate_point(value):
-    """ Dehydrator for Point data.
-
-    :param value:
-    :type value: Point
-    :return:
-    """
-    return _hydration.dehydrate_point(value)
-
-
```

Comment:
Because I'm a derp :sweat_smile: 

That was a copy pasta mistake. The exact same code can be found right above it https://github.com/neo4j/neo4j-python-driver/pull/767/files/a2b0ef051806e05cb2b2cf5c63a31e318dd51c69#diff-0260b4d7c5791c8c36f46ad24ae30e20fff2a48958b9c918b59be77428be9864L59-L72

Reply:
I marked the files which I reviewed as viewed, so the link you send is not working properly. Could you tell me the name of the file? (because, the lines I can guess by the line, L59-L72)

Reply:
`neo4j/spatial/__init__.py` (the same file, right above the code you highlighted)